spec/features/commands_spec.rb

```diff
@@ -234,6 +237,56 @@
     end.to change { deployment_requests.count }.by(1)
   end
 
+  scenario 'stealing a lock by action' do
+    command '/deploy lock staging on acme-inc/api', as: slack_accounts(:david)
+    expect(command_response.message).to eq Slack::Message.new(text: 'Locked *staging* on acme-inc/api')
+
+    command '/deploy lock staging on acme-inc/api', as: slack_accounts(:david)
+    expect(command_response.message).to eq Slack::Message.new(text: '*staging* is already locked')
+
+    expect(SecureRandom).to receive(:uuid).and_return('a1a111a1-1111-1a1a-a1a1-111aaa111111').at_least(:once)
+    command '/deploy lock staging on acme-inc/api', as: slack_accounts(:steve)
+    expect(command_response.message).to eq Slack::Message.new(text: "*staging* was locked by <@david> less than a minute ago.\nYou can steal the lock with `/deploy lock staging on acme-inc/api!`.", attachments: steal_lock_attachments)
```

Comment:
One minor thing. Can we remove `You can steal the lock with `/deploy lock staging on acme-inc/api!` from these messages? I think we should keep the / command for lock stealing, but the canonical method should be to just click the button.